applications/DEMApplication/python_scripts/dem_default_input_parameters.py

```diff
@@ -77,6 +77,14 @@ def GetDefaultInputParameters():
             "CoordinationNumber"               : 10,
             "LocalCoordinationNumberOption"    : false,
             "GlobalCoordinationNumberOption"    : true,
+            "test_number" : 1,
+            "inner_mesh_diameter" : 0.00015,
+            "outer_mesh_diameter" : 0.00015,
+            "inner_skin_factor" : 2.4,
+            "outer_skin_factor" : 0.8,
+            "use_gid_skin_mesh" : true,
```

Comment:
"respect_preprocess_marked_skin" (It could come from other pre-processors, not GiD)

Reply:
Done!